core/types/access_list_tx.go

```diff
@@ -554,3 +554,11 @@ func (tx *AccessListTx) Sender(signer Signer) (libcommon.Address, error) {
 	tx.from.Store(addr)
 	return addr, nil
 }
+
+func (tx *AccessListTx) PutOptions(options *types2.TransactionConditions) {
+	tx.TransactionConditions = options
+}
+
+func (tx *AccessListTx) GetOptions() *types2.TransactionConditions {
+	return tx.TransactionConditions
+}
```

Comment:
as the logic is trivial now, do you still need these methods, or is it ok to access the field directly (and remove the methods)?

if you prefer to keep it, it should be renamed to `Get/PutTransactionConditions()`